src/_includes/partials/components/brand.njk

```diff
@@ -7,9 +7,9 @@
     {% endif %}
 
     <a class="brand__desktop" rel="home" href="{{ homeUrl }}"{% if page.url === homeUrl %} aria-current="page"{% endif %}>
-        {%- svg_sprite "logo-ildh-wordmark" -%}
+        {%- svg_sprite "logo-ildh-wordmark", translations[lang].brandAltText, false -%}
```

Comment:
Do you have to pass in all the arguments? If you do, it's probably better to either pass null as the final argument or for clarity the actual heading selector "h2".

Reply:
The purpose of these args, in this particular case, are to provide the alt text for the SVG and to indicate that this SVG in particular should not have `aria-hidden` set to true, or the home link would not be perceptible by AT's as anything other than a blank link. I'd chosen `false` as it's the opposite of `true`, but `null` is just as effective and I have no objections to switching to that if you'd like :)

Reply:
I seem to have made a mistake on what this short code was for, so you can ignore my comment. Sorry about that.